src/test/scala/reactive/MainRoutingSpec.scala

```diff
@@ -0,0 +1,78 @@
+package reactive
+
+import akka.http.scaladsl.model.StatusCodes.{ NoContent, OK, SwitchingProtocols }
+import akka.http.scaladsl.model.headers.{ CustomHeader, Upgrade, UpgradeProtocol }
+import akka.http.scaladsl.model.ContentTypes.`application/json`
+import akka.http.scaladsl.model.ws.{ Message, UpgradeToWebsocket }
+import akka.http.scaladsl.model.{ HttpRequest, HttpResponse, StatusCodes }
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import akka.stream.FlowMaterializer
+import akka.stream.scaladsl.Flow
+import akka.util.Timeout
+import org.scalatest.{ FlatSpec, Matchers }
+import reactive.tweets.domain.{ Tweet, User }
+import reactive.tweets.marshalling.TweetJsonProtocol
+
+import scala.concurrent.duration.DurationInt
+import scala.language.postfixOps
+
+class MainRoutingSpec extends FlatSpec with Matchers with ScalatestRouteTest with TweetJsonProtocol {
+  implicit val timeout = Timeout(1000 millis)
+
+  "Main" should "serve the index page on /" in {
+    Get("/") ~> Main.mainFlow ~> check {
+      status shouldBe OK
```

Comment:
Conscious decision to skip that. We could add it if we agree it is absolutely necessary.